pkg/apiserver/registry/system/supportbundle/rest.go

```diff
@@ -172,6 +172,9 @@ func (r *supportBundleREST) New() runtime.Object {
 	return &systemv1beta1.SupportBundle{}
 }
 
+func (r *supportBundleREST) Destroy() {
+}
```

Comment:
there may be something to do here (ideally) since we have some internal state in `supportBundleREST` (`cache`)?

Reply:
Based on Quan's comment above, looks like there is no real need for `Destroy` anywhere, so please disregard this